Dockerfile

```diff
@@ -11,7 +11,7 @@ ENV CHOKIDAR_USEPOLLING true
 
 VOLUME /var/lib/eg
 
-RUN yarn global add express-gateway@$EG_VERSION
+RUN yarn global add express-gateway@$EG_VERSION --ignore-optional
```

Comment:
I asked this before, but I'm not sure I got a response.  Why are we using `yarn` instead of `npm`?

Reply:
Ah, okay.  You can change the global path in NPM, though.  `npm config set prefix "/chicken/noodle/soup"`